### PR TITLE
Support PATHS in IMPORT keyword

### LINKED ISSUE #2599: IMPORT does not support PATHS
Variables from `PATHS` (path variables starting with `$`) should be support in `IMPORT`, similarly to `INCLUDE`.

### PR DESCRIPTION
**Pull request by asnyv**
Closes #2599

Utilizes the function used for `INCLUDE` to process the path given to `IMPORT`.
Also added the scenario to one of the existing tests

### CODE REVIEW

File: src/opm/parser/eclipse/Parser/Parser.cpp [hunk 1]
```diff
@@ -776,10 +776,10 @@ RawKeyword * newRawKeyword( const std::string& deck_name, ParserState& parserSta
 }
 
 // This code is made to skip part of the eclipse file containing the contents of UDT
-// UDT is currently not a asupported keyword. 
+// UDT is currently not a asupported keyword.
```
Comment (akva2) [inline src/opm/parser/eclipse/Parser/Parser.cpp:779]: since you touch the line maybe you can fix the typo while at it?

Comment (asnyv) [reply src/opm/parser/eclipse/Parser/Parser.cpp:779]: sorry, was my editor that removed the trainling spaces. Just realized. I'm planning to remove all the trailing spaces to not touch all these files, but I can fix this typo if you want me to.

Comment (akva2) [reply src/opm/parser/eclipse/Parser/Parser.cpp]: right. in that case let's just deal with it separate from this PR to keep the changes minimal.

Comment (akva2) [reply src/opm/parser/eclipse/Parser/Parser.cpp]: https://github.com/OPM/opm-common/pull/2601 takes care of this

**Comment by akva2** [comment]
jenkins build this please

File: src/opm/parser/eclipse/Parser/Parser.cpp [hunk 2]
```diff
@@ -792,7 +792,7 @@ void skipUDT( ParserState& parserState, const Parser& parser) {
       if (record_count < count_max)
           record_count++;
       else {
-           std::string deck_name = str::make_deck_name( line );        
+           std::string deck_name = str::make_deck_name( line );
```
Comment (asnyv) [inline src/opm/parser/eclipse/Parser/Parser.cpp:795]: apparently still got it wrong 😒

Comment (akva2) [reply src/opm/parser/eclipse/Parser/Parser.cpp:795]: git add -p is one easy way to deal with it.

**Comment by asnyv** [comment]
sorry for the noise, think this should be ok @akva2? Next time I'll turn off the autoformatting 😅

File: src/opm/parser/eclipse/Parser/Parser.cpp (hunk 2 above)
Comment (asnyv) [reply src/opm/parser/eclipse/Parser/Parser.cpp]: yes ok, thought it was going to be smashed anyways, but I'll rebase then 😊

**Comment by akva2** [comment]
purrfect :)

File: src/opm/parser/eclipse/Parser/Parser.cpp (hunk 2 above)
Comment (asnyv) [reply src/opm/parser/eclipse/Parser/Parser.cpp]: Screw-ups cleaned 🛀

**Comment by bska** [comment]
jenkins build this please

**Pull request merged**